docs/getting_started/tutorial.md

```diff
@@ -31,12 +27,12 @@ If none of the preceding commands return a version number, or return a version l
 ### Create and activate a virtual environment
 
 This tutorial recommends using a virtual environment, which isolates installed dependencies from other projects.
-This tutorial uses [`venv`](inv:python#tutorial/venv), which is packaged with Python 3. On Ubuntu, it may be necessary to run `sudo apt install python3-venv` to install it.
+This tutorial uses [`venv`](inv:python#tutorial/venv), which is packaged with uv.
```

Comment:
Again, in here there's no link to `uv`, and linking to the docs for `venv` is misleading because they're different. How are people supposed to know what `uv` is and how to install it, and why is it a requirement to use Wagtail?